sklearn/utils/_unittest_backport.py

```diff
@@ -149,7 +149,7 @@ def __exit__(self, exc_type, exc_value, tb):
 
 
 class TestCase(unittest.TestCase):
-    longMessage = False
+    longMessage = True
```

Comment:
This was set to False in #9536, but [`longMessage` is True for all `unittest.TestCase` by default.](https://docs.python.org/3/library/unittest.html#unittest.TestCase.longMessage) so I believe that was erroneous.

Reply:
I don't understand your comment. If it's true by default, why would someone set it to True?